app/hooks/use-realtime-price.ts

```diff
@@ -0,0 +1,22 @@
+import { WatchQueryFetchPolicy } from "@apollo/client"
+import { useDisplayCurrencyQuery, useRealtimePriceQuery } from "@app/graphql/generated"
+import { useIsAuthed } from "@app/graphql/is-authed-context"
+
+export const useRealtimePriceWrapper = (
+  { fetchPolicy }: { fetchPolicy: WatchQueryFetchPolicy } = {
+    fetchPolicy: "network-only",
```

Comment:
I don't think the default here should be `network-only` as this hook will render multiple times on every single screen and if we ever use `network-only` itll result in a ton of network calls.

Reply:
Maybe there should be no default?

It's used in 4 places, and 3 of them should be network only. The one which is inside another hook and can be called with high frequency is cache-first. So logically this should be good but maybe this is not obvious from the function signature. Maybe I can have no default and let the caller explicitly say which network policy to apply?